components/speeddial/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { StyledSpeedDial } from './StyledSpeedDial'
+import {
+    SpeedDial as MuiSpeedDial,
+    SpeedDialIcon,
+    SpeedDialAction
+} from '@mui/material'
+import { RateReview, Message } from '@mui/icons-material'
+
+import { showDrawer } from '@/lib/redux/slices/drawer'
+import { useDispatch } from 'react-redux'
+import { SpeedDialProps } from '@/utils/types/components/speed-dial'
+import { DRAWER_TYPES } from '@/utils/constants'
+
+const { TESTIMONIAL, CLIENT_REQUEST } = DRAWER_TYPES
+const SpeedDialOptions: SpeedDialProps[] = [
+    { icon: <Message />, label: 'Contact Me', content: CLIENT_REQUEST },
+    {
+        icon: <RateReview />,
+        label: 'Create a Testimonial',
+        content: TESTIMONIAL
+    }
+]
+const SpeedDial = () => {
+    const dispatch = useDispatch()
+
+    const drawerToggle = (content: string) => dispatch(showDrawer(content))
```

Comment:
Rename the function to how i said, `onDrawerToggle`.

Reply:
Add label as the 2nd param in the function here. Switch back to an object being sent into `showDrawer`:
```
dispatch(showDrawer({ content, title: label }))
```